### PR TITLE
fix: Use docker plugin to have gradle build ssr catalogue image

### PR DESCRIPTION
**Pull request by connoratrug**
Build a docker images for the ssr-catalogue ( for deployment) 

if setup is done add to circle CI config 
```      # push ssr-catalogue to repo
      - run: ./gradlew dockerPush

### CODE REVIEW

File: apps/build.gradle [hunk 1]
```diff
@@ -57,15 +57,17 @@ subprojects {
         delete "${project.parent.buildDir}/resources/main/public_html/apps/${project.name}"
     }
 
-    //copy styleguide or normal app
+    //copy nuxt-ssr, styleguide or normal app
     if(project.name == 'nuxt-ssr') {
```
Comment (mswertz) [inline apps/build.gradle:61]: why does nuxt need seperate copy, can't you use copy we use for all other apps?

**Comment by mswertz** [comment]
I also think you need to add this module to apps/package.json

File: apps/build.gradle (hunk 1 above)
Comment (connoratrug) [reply apps/build.gradle:61]: it has a different folder structure

**Comment by connoratrug** [comment]
> I also think you need to add this module to apps/package.json

added

**Review by mswertz — APPROVED** [review]
untested but looks good

**Comment by connoratrug** [comment]
grrrrr. https://status.circleci.com/

**Pull request merged**